administration/templates/administration/overview.html

```diff
@@ -14,7 +14,7 @@
 
 {% block html_head %}
     {# This template loads in either the dev or prod Vue library, depending on settings.DEBUG #}
```

Comment:
Missing ``{{ block.super }}``

Not required atm, but in the future omitting that might cause problems